xe/thinCLIProtocol.cs

```diff
@@ -258,6 +257,7 @@ public static class Types
 
     public static class Messages
     {
+        private const string MAGIC_STRING = "XenSource thin CLI protocol";
```

Comment:
I think this should be called something that indicates what it is not just magic string